docker-compose.yaml

```
@@ -43,5 +43,34 @@ services:
       start_period: 10s
 
+  app:
+    build:
+      context: ./apps/web
+      dockerfile: Dockerfile
+    ports:
+      - "3000:3000"
+    environment:
+      # Database
+      DATABASE_URL: "postgresql://opencut:opencutthegoat@db:5432/opencut"
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)